python/ray/train/lightning/_lightning_utils.py

```diff
@@ -0,0 +1,36 @@
+from typing import Iterable, Type
+
+from pytorch_lightning import LightningDataModule
+
+from ray.air import session
+
+if TYPE_CHECKING:
+    from ray.data.dataset import Dataset
+
+
+def process_datasets(
+    train_dataset: "Dataset",
+    val_dataset: "Dataset",
+    test_dataset: "Dataset",
+    predict_dataset: "Dataset",
+) -> LightningDataModule:
+    """Convert Ray dataset shards to a PTL DataModule."""
+    # TODO (s10a): batch_size
+    torch_datasets = {"train_dataset": train_dataset.to_torch()}
```

Comment:
we shouldn't use to_torch but instead use `iter_torch_batches`

Reply:
Can you explain the difference? I though `to_torch` is basically a shortcut to convert `iter_torch_batches` to an IterableDataset (which we want since need PyTorch datasets to create PyTorch dataloaders which can be used by LightningDataModule)

Reply:
Hi @sumanthratna, i think we are deprecating `to_torch` and `iter_torch_batches` is more preferred to the RAY AIR design.